modules/gdscript/gdscript_function.cpp

```diff
@@ -1093,6 +1093,8 @@ Variant GDScriptFunction::call(GDScriptInstance *p_instance, const Variant **p_a
 								OPCODE_BREAK;
 							}
 						}
+					} else if (methodstr == "call_recursive") {
+						methodstr = String(*argptrs[0]);
```

Comment:
While I prefer to not add exceptions, I think in this case it can be done here. But it should be clear that it's the correct thing and give a good error message. Something like this:

```suggestion
					} else if (methodstr == "call_recursive" && basestr == "TreeItem") {
						if (argc >= 1) {
							methodstr = String(*argptrs[0]) + " (via TreeItem.call_recursive)";
							if (err.error == Callable::CallError::CALL_ERROR_INVALID_ARGUMENT) {
								err.argument += 1;
							}
						}
```

Reply:
@vnen  Actually, `String(*argptrs[0]) + " (via TreeItem.call_recursive)"` results in -> `'called_method (via TreeItem.call_recursive)'`, when it should be `'called_method' (via TreeItem.call_recursive)`. 
It's because of `err_text = _get_call_error(err, "function '" + methodstr + "' in base '" + basestr + "'", (const Variant **)argptrs);` at line 1097. 
I'm wondering if there's a way to ignore/escape the leading `'` from `"' in base '"`
Or the other way would be to remove `(via TreeItem.call_recursive)` from the error message.

Reply:
> Actually, `String(*argptrs[0]) + " (via TreeItem.call_recursive)"` results in -> `'called_method (via TreeItem.call_recursive)'`, when it should be `'called_method' (via TreeItem.call_recursive)`.

That's a minor detail, I don't think it's an issue (and the same should already happen with `call()`). I prefer to make it clear it's via `via call_recursive` because the method name might be passed via variable, so it might not be obvious to the user what it is just by looking at the break point.

Adding to `_call_recursive_bind` would be ideal since it doesn't require an exception in GDScript for it, but I don't think it can be done while showing a good error message.

Reply:
@vnen Cool! Then I'd make the commit as you've suggested earlier, adding the `'(via TreeItem.call_recursive)'` to the GDScript function.